### DISCUSSION TITLE
Some beta testing feedback for this amazing server and app

### DISCUSSION THREAD

**Discussion by cfinleyca** [advplyr/audiobookshelf-app#509] [General]
1st thing: Thanks so much for your time and care in making this awesome open source wonder.  I am posting here to ensure that these issues are valid and make sense. This is my first dip into open source and I want to be respectful. I am happy to write issues for any that seem worthy. Thanks again for your time and the incredible job with AudioBookShelf. Thanks again. Here is my feedback. 

IOS APP Version 0.9.60 (17) IOS 16.1.1 iPhone 13 Pro
UX Enhancements
* Track Scrubbing
	* Description: Scrubbing over the whole book is unusual as a default and frustratingly hard to control over most any book. The UI for the Chapter/Track scrubbing control was only found by searching the GitHub issues
	* Recommendation: Setting default to track scrubbing. Additionally, suggesting wording change  “Chapter Track” to a toggle menu item to something like “Scrub Track/Scrub Book”
* Selectable items (links) not obvious
	* Description: The links on the Detail View are not obvious (found them by accident, but very convenient)
	* Recommendation: highlight the items with a color or text change like links (doesn’t have to be link looking but maybe a color highlight like search does)
* Links (like author, series) don’t have breadcrumbs for returning to previous view
	* Description: While viewing book details if you select a link (e.g. Author Brandon Sanderson) you are taken to an awesome list of all of Brandon’s books but if you want to get back to the book you were looking at, you are forced to reopen search (thanks for remembering my search terms) or by finding the book in the list and selecting it again. It gets worse if you select the series link because you are taken to a list of the books in the series but now you get an arrow that looks like it takes you back to your detail View but alas it takes you to the broken Series View showing no books (See bugs). 
	* Recommendation: Use the arrow like on series but to follow the breadcrumbs back to the Detail View (this precedent has be set in the account, user stats, downloads interactions)
* Search terms could use a clear button
	* Description: after typing in a search you select the details you want to see and you return to search your search terms are saved as are your results (this is awesome) but for a new search, having to delete by character for all of the search terms seems counterintuitive.
	* Recommendation: The addition of a clear search button (tiny X, tiny “Clear” at the right side of the search field) would make it so easy to clear a previous search. 
* Lock Player confusion
	* Description: I have turned it on still don’t know what it does. After some investigation it looks like it prevents scrubbing. 
	* Recommendation: a description or maybe more descriptive name like “Lock Scrubbing”
* Close Player could use an X icon

Potential Bugs
* Headphone pause not working when playback rate != 1X
	* To recreate this:
		* Using headphones with play/pause control, change playback speed to anything other than 1X, push the play button to start playback, now push the button again to pause (typically backs up a few seconds and resumes, which makes me think it’s actually calling play again) 
* Series Display does not show books when all series are listed
	* To recreate this:
		* Click the series tab button the display shows the number of series in the library but nothing showing in the list. However, they do display when selecting a series link from the book detail view. But even then if you click the arrow returning you to the Series Tab View it shows no series just the number of series sets in the library.

**Comment by hobesman** [comment]
I like the suggestions. The bugs you mentioned don't happen for me though. Are you seeing those on mobile or on the browser?

**Reply by cfinleyca** [reply]
These are only on the IOS app.

**Comment by advplyr** [comment]
Thanks for the feedback. I've updated the search input to be clearable for the next release.
